tools/crm_simulate.c

```diff
@@ -271,27 +271,27 @@ static GOptionEntry operation_entries[] = {
       "Show utilization information",
       NULL },
     { "profile", 'P', 0, G_OPTION_ARG_FILENAME, &options.test_dir,
-      "Run all tests in the named directory to create profiling data",
-      NULL },
+      "Process all the XML files in the named directory to create profiling data",
+      "DIR" },
     { "repeat", 'N', 0, G_OPTION_ARG_INT, &options.repeat,
       "With --profile, repeat each test N times and print timings",
       "N" },
     { "pending", 'j', 0, G_OPTION_ARG_NONE, &options.print_pending,
-      "Display pending state if 'record-pending' is enabled",
+      "Display pending state if the 'record-pending' property is 'true' on an operation",
```

Comment:
Oops, we missed something when this was converted to GOption.

We switched this to always true a while back and made this option hidden. When we switched to GOption it stopped being hidden. It should be a hidden no-op and commented as deprecated.

Reply:
Fixed.